phys2bids/phys2bids.py

```diff
@@ -35,6 +35,7 @@
 
 from phys2bids import utils, viz, _version
 from phys2bids.bids import bidsify_units, use_heuristic, participants_file
+from phys2bids.bids import dataset_description_file, README_file
```

Comment:
At this point, we're importing a lot from bids. It would be better to add "bids" to the import of phys2bids (two lines above) and update the code consequently!

Reply:
done